CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning]
+(https://semver.org/spec/v2.0.0.html).
+
+## [4.6.2] - 2023-03-08
```

Comment:
We should also update the `unity/PitayaExample/LibPitaya.nuspec` file to keep the version consistent.